apps/frontend/src/pages/Send/components/TransactionDetails.tsx

```diff
@@ -86,11 +90,34 @@ const TransactionDetails = ({
       <AddSignatureFromQrCode
         importSignatureFromFile={() => {}}
         psbt={finalPsbt}
-        currentBitcoinPrice={currentBitcoinPrice}
+        closeModal={closeModal}
       />
     );
   };
 
+  const adjustInputs = async (utxos: UTXO[]) => {
+    const psbt = await createTransactionAndSetState!(
+      [
+        {
+          address: finalPsbt.txOutputs[0].address!,
+          value: Number(finalPsbt.txOutputs[0].value.toString())
+        }
+      ],
+      0,
```

Comment:
This will reset the transaction fee and fee rate if a custom one was selected by the user.

We should preserve the selected feeRate if possible.